charts/grafana/README.md

```diff
@@ -217,6 +217,7 @@ This version requires Helm >= 3.1.0.
 | `imageRenderer.priorityClassName`          | image-renderer deployment priority class                                           | `''`                             |
 | `imageRenderer.service.portName`           | image-renderer service port name                                                   | `'http'`                         |
 | `imageRenderer.service.port`               | image-renderer service port used by both service and deployment                    | `8081`                           |
+| `iamgeRenderer.grafanaSubPath`             | Grafana sub path to use for image renderer callback url                            | `''`                             |
```

Comment:
Should it be `imageRender.grafanaSubPath` ?

Reply:
Ah, thank you for noticing